tests/integration/test_cli.py

```diff
@@ -36,3 +36,16 @@ def test_stack_trace_printed_on_error(local_app):
     assert rc == 2
     assert 'Traceback' in stderr
     assert 'foobarbaz' in stderr
+
+
+def test_debug_flag_enables_logging(local_app):
+    req_file = os.path.join(local_app, 'requirements.txt')
+    with open(req_file, 'w') as f:
+        f.write('pip==10.0.0\n')
+    p = subprocess.Popen(['chalice', '--debug', 'package', 'out'],
```

Comment:
So the coverage is down because it is being ran in a subprocess?

Reply:
Yes unless i'm missing something I cant see how to even test top level args with the click test runner.

Reply:
Does just passing the `cli` function to the click runner not work?